src/main/java/control/ToolViewController.java

```diff
@@ -36,10 +34,10 @@ public ToolViewController(ControllerManager controllerManager) {
      * the event handlers.
      */
     private void initializeTools() {
-        cameraBlockCreationTool = new ToolButton("Add camerashot",
+        new ToolButton("Add camerashot",
```

Comment:
If they are needed in the future we can add them in the future. At this moment, it is bad practice to have unused variables. Unless someone is going to tell me that they are using these variables at this very moment while implementing new features